{"name": "Проверка отображения заказов пользователя в разделе 'История заказов'", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'", "trace": "request = <SubRequest 'setup' for <Function test_order_details_popup[firefox]>>\n\n    @pytest.fixture(params=[\"chrome\", \"firefox\"], scope=\"class\")\n    def setup(request):\n>       driver = initialize_driver(request.param)\n\nconftest.py:6: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nbrowser_name = 'firefox'\n\n    def initialize_driver(browser_name):\n        if browser_name == \"chrome\":\n            driver = webdriver.Chrome(ChromeDriverManager().install())\n        elif browser_name == \"firefox\":\n>           driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())\nE           TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n\nutils\\driver_setup.py:9: TypeError"}, "parameters": [{"name": "setup", "value": "'firefox'"}], "start": 1730908181270, "stop": 1730908181270, "uuid": "0006ab79-3fdb-46c9-bb21-02d8ce026784", "historyId": "3f78e440f3fd1c78d9848e8775b89e62", "testCaseId": "6c1ab92dd29e510d43e022d239463f4e", "fullName": "tests.test_order_feed.TestOrderFeed#test_user_orders_displayed_in_order_feed", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_feed"}, {"name": "subSuite", "value": "TestOrderFeed"}, {"name": "host", "value": "WorkPC"}, {"name": "thread", "value": "2056-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_feed"}]}